### PR TITLE
Defined all Metasprite data

### PR DESCRIPTION
**Pull request by evie-calico**
I got bored and wanted to finish since I need this for my project, so a lot is defined as "Unknown", or "Part2Unknown" (Since I started on the second half). Should be much easier to go through now if one needs to edit the metasprites, should save someone the days it took me in the future.

### CODE REVIEW

File: bank_00b.asm [hunk 1]
```diff
@@ -1,12 +1,3286 @@
 SECTION "ROM Bank $00b", ROMX[$4000], BANK[$b]
 
-INCBIN "baserom.gb", $2c000, $2d8b9 - $2c000
+;Byte 1 and 2 are some sort of X and Y offsets for the Metasprite.
+;Byte 3 is the tile used by the Metasprite.
+;Byte 4 is the attributes, listed below
+;7: Render priority
+;6: Y flip
+;5: X flip
+;4: End of Metasprite
```
Comment (huderlem) [inline bank_00b.asm]: Thanks for doing this dump.

However, please amend this data structure description, as it's incorrect.  Each metasprite entry is simply a raw OAM entry, where bit 0 of the last byte marks the end of the metasprite.  https://bgb.bircd.org/pandocs.htm#vramspriteattributetableoam

Comment (evie-calico) [reply bank_00b.asm]: Sorry, I changed the bit for my own project and I guess I forgot to change it in the description.

**Comment by GreenAndEievui** [comment]
I changed it to say unused, although that might not be entirely accurate. A few of the metasprites have bit 4 set, but I can't find where it's used.

**Comment by GreenAndEievui** [comment]
The description should be accurate now.

**Comment by evie-calico** [comment]
Oops, I uploaded the wrong file twice. The description should _actually_ be accurate now.

**Pull request merged**